sdk/src/main/java/com/horizen/account/api/rpc/types/DebugTraceBlockByIdView.java

```diff
@@ -0,0 +1,16 @@
+package com.horizen.account.api.rpc.types;
+
+import com.fasterxml.jackson.annotation.JsonView;
+import com.horizen.evm.interop.EvmResult;
+import com.horizen.serialization.Views;
+
+import java.util.Arrays;
+
+@JsonView(Views.Default.class)
+public class DebugTraceBlockByIdView {
+    public Object[] debugTraceTransactionViews;
```

Comment:
Why not use `DebugTraceTransactionView` instead of `Object` and add a type cast below?